### PR TITLE
Updated documentation for OshiSysMonitor

### PR DESCRIPTION
**Pull request by hardikbajaj**
Fixes #XXXX.

### Description

Updates documentation for OshiSysMontior. Accidentally https://github.com/apache/druid/pull/14799 got closed which was initial PR and has all the details, modified all suggestions here

#### Release note

<hr>

##### Key changed/added classes in this PR
 * `MyFoo`
 * `OurBar`
 * `TheirBaz`

<hr>

This PR has:

- [ ] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] a release note entry in the PR description.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [ ] been tested in a test Druid cluster.

### CODE REVIEW

File: docs/operations/metrics.md [hunk 1]
```diff
@@ -427,6 +432,51 @@ These metrics are only available if the `SysMonitor` module is included.
...
+|------|-----------|----------|------------|
+|`sys/swap/free`|Free swap||Varies|
+|`sys/swap/max`|Max swap||Varies|
+|`sys/swap/pageIn`|Paged in swap||Varies|
+|`sys/swap/pageOut`|Paged out swap||Varies|
+|`sys/disk/write/count`|Writes to disk|`diskName`|Varies|
+|`sys/disk/read/count`|Reads from disk|`diskName`|Varies|
+|`sys/disk/write/size`|Bytes written to disk. One indicator of the amount of paging occurring for segments.|`diskName`|Varies|
+|`sys/disk/read/size`|Bytes read from disk. One indicator of the amount of paging occurring for segments.|`diskName`|Varies|
+|`sys/disk/queue`|Disk queue length. Measures number of requests waiting to be processed by Disk|`diskName`|Generally 0|
```
Comment (ektravel) [inline docs/operations/metrics.md:449]:
```suggestion
|`sys/disk/queue`|Disk queue length. Measures number of requests waiting to be processed by disk|`diskName`|Generally 0|
```

File: docs/operations/metrics.md [hunk 2]
```diff
@@ -427,6 +432,51 @@ These metrics are only available if the `SysMonitor` module is included.
...
+|`sys/net/read/size`|Bytes read from the network|`netName`, `netAddress`, `netHwaddr`|Varies|
+|`sys/net/read/packets`|Total packets read from the network|`netName`, `netAddress`, `netHwaddr`|Varies|
+|`sys/net/write/packets`|Total packets written to the network|`netName`, `netAddress`, `netHwaddr`|Varies|
+|`sys/net/read/errors`|Total network read errors|`netName`, `netAddress`, `netHwaddr`|Generally 0|
+|`sys/net/write/errors`|Total network write errors|`netName`, `netAddress`, `netHwaddr`|Generally 0|
+|`sys/net/read/dropped`|Total packets dropped coming from network|`netName`, `netAddress`, `netHwaddr`|Generally 0|
+|`sys/net/write/collisions`|Total network write collisions|`netName`, `netAddress`, `netHwaddr`|Generally 0|
+|`sys/fs/used`|Filesystem bytes used |`fsDevName`, `fsDirName`|< max|
+|`sys/fs/max`|Filesystem bytes max |`fsDevName`, `fsDirName`|Varies|
+|`sys/fs/files/count`|Filesystem total IO Nodes |`fsDevName`, `fsDirName`|< max|
```
Comment (ektravel) [inline docs/operations/metrics.md:461]:
```suggestion
|`sys/fs/files/count`|Filesystem total IO nodes |`fsDevName`, `fsDirName`|< max|
```

File: docs/operations/metrics.md [hunk 3]
```diff
@@ -427,6 +432,51 @@ These metrics are only available if the `SysMonitor` module is included.
...
+|`sys/fs/files/free`|Filesystem free IO nodes|`fsDevName`, `fsDirName`| Varies |
+|`sys/mem/used`|Memory used||< max|
+|`sys/mem/max`|Memory max||Varies|
+|`sys/mem/free`|Memory free||Varies|
+|`sys/storage/used`|Disk space used|`fsDirName`|Varies|
+|`sys/cpu`|CPU used|`cpuName`, `cpuTime`|Varies|
+|`sys/uptime`|Total system uptime||Varies|
+|`sys/la/{i}`|System CPU Load averages over past i minutes, where i={1,5,15}||Varies|
```
Comment (ektravel) [inline docs/operations/metrics.md:469]:
```suggestion
|`sys/la/{i}`|System CPU load averages over past `i` minutes, where `i={1,5,15}`||Varies|
```

**Review by ektravel — COMMENTED** [review]
Thank you for making these changes, @hardikbajaj. Left a few suggestions.

File: docs/operations/metrics.md [hunk 4]
```diff
@@ -404,7 +406,10 @@ The following metric is only available if the `EventReceiverFirehoseMonitor` mod
 |`ingest/events/buffered`|Number of events queued in the `EventReceiverFirehose` buffer.|`serviceName`, `dataSource`, `taskId`, `taskType`, `bufferCapacity`|Equal to the current number of events in the buffer queue.|
 |`ingest/bytes/received`|Number of bytes received by the `EventReceiverFirehose`.|`serviceName`, `dataSource`, `taskId`, `taskType`|Varies|
 
-## Sys
+## Sys [Deprecated]
+
+> SysMonitor is now deprecated and will be removed in future releases.
```
[the remark this answers is not in the document]
Comment (hardikbajaj) [reply docs/operations/metrics.md:411]: Sure, should I add it in ::: info wrapper instead like [here](https://github.com/apache/druid/pull/14912/files#diff-4153c0d20e748deebfabfa5f90696f686d98ff83362e7fea73937a025ed42db5L39) . I've seen some deprecation notices in this repo put this way

[the remark this answers is not in the document]
Comment (ektravel) [reply docs/operations/metrics.md:411]: Yes, please. Let's use the special admonitions syntax:

```suggestion
:::info
SysMonitor is now deprecated and will be removed in future releases. Instead, use the new OSHI monitor called [OshiSysMonitor](#oshisysmonitor). The new monitor has a wider support for different machine architectures including ARM instances.
:::
```

**Review by abhishekagarwal87 — APPROVED** [review]

**Pull request merged**